privilege/privileges/cache.go

```diff
@@ -462,15 +467,43 @@ func (p *MySQLPrivilege) loadTable(sctx sessionctx.Context, sql string,
 	}
 }
 
+// parseHostIPNet parses an IPv4 address and its subnet mask (e.g. `127.0.0.0/255.255.255.0`),
```

Comment:
from on-call group, it seems some customer use ipv6 host conf- -

Reply:
Sure

Reply:
@lysu comment addressed ,PTAL